app/Resources/amp/amp-theme/index.html.twig

```diff
@@ -0,0 +1,60 @@
+{#
+Available variables:
+- object - Object of an class instance which you provided in the bundle config. e.g. AppBundle\Entity\Post
+#}
+<!doctype html>
```

Comment:
i would add twig blocks around most important pars - so templators can extend that template and change only needed parts.